src/kolibri_gnome/kolibri_context.py

```diff
@@ -211,6 +217,13 @@ def parse_x_kolibri_app_url_tuple(self, url_tuple: SplitResult) -> str:
         """
         return url_tuple._replace(scheme="", netloc="").geturl()
 
+    def __webkit_web_context_on_download_started(
+        self, webkit_web_context: WebKit2.WebContext, download: WebKit2.Download
+    ):
+        url = download.get_request().get_uri()
```

Comment:
Nothing is done with `url` here. Please remove it.

Reply:
Ah, I see `url` used in upcoming commits! For the sake of moving things forward let's not rebase. I was talking about 2nd commit "Add support for file downloads in KolibriContext" where url is assigned but not used.